build.gradle

```diff
@@ -303,7 +303,7 @@ android {
     abi {
       enable true
       reset()
-      include 'armeabi-v7a', 'arm64-v8a'
+      include 'armeabi-v7a', 'arm64-v8a', 'x86'
```

Comment:
NB: This should be reverted before merging.

Reply:
OK, I'll leave it in a new line commented with a note that you have to enable it in order to test the app in a virtual device with Android 10 locally

Reply:
Done.